src/pages/academia.js

```diff
@@ -86,12 +162,106 @@ export default () => {
+                        </ArticleH2>
+                        <BodyText>
+                            Access our MongoDB for Academia community to
+                            collaborate, share tips and get inspired.
+                        </BodyText>
+                    </OfferingsContent>
                 </div>
+                {!isMobile && (
+                    <img src={TeachMongoDBImage} alt="" width="550px" />
+                )}
+            </BodyContent>
+
+            <EligibilitySection>
+                <BodyContent>
+                    <div>
+                        <EligibilityContent>
+                            <StyledSectionHeader>
+                                For Educators
+                            </StyledSectionHeader>
+                            <BodyText>
+                                If you’re a student, you can apply for the{' '}
+                                <StyledLink href="https://education.github.com/pack">
+                                    GitHub Student Developer Pack
+                                </StyledLink>{' '}
+                                and get access to MongoDB Atlas, University
+                                on-demand content and certifications.
+                            </BodyText>
+                        </EligibilityContent>
+
+                        <EligibilityContent>
+                            <StyledSectionHeader>
+                                For Students
+                            </StyledSectionHeader>
+                            <BodyText>
+                                MongoDB for Academia is for educators who want
+                                to prepare students for careers that require
+                                in-demand database skills that power modern
+                                applications.
+                                <StyledP>
+                                    You’re eligible for this program if you
+                                    teach:
+                                </StyledP>
+                                <StyledBullet>
+                                    <li>
+                                        <StyledSpan>
+                                            Higher Education, College and
+                                            University programs
+                                        </StyledSpan>
+                                    </li>
+                                    <li>
+                                        <StyledSpan>
+                                            Bootcamps and Online Courses
+                                        </StyledSpan>
+                                    </li>
+                                </StyledBullet>
+                            </BodyText>
+                        </EligibilityContent>
+                    </div>
+
+                    {!isMobile && <img src={AcademiaLeafImage} alt="" />}
```

Comment:
Let's see if we can push this image down a bit, since it looks to belong in the For Students section